rpc/src/v1/traits/traces.rs

```diff
@@ -43,6 +46,7 @@ pub trait Traces: Sized + Send + Sync + 'static {
 		delegate.add_method("trace_transaction", Traces::transaction_traces);
 		delegate.add_method("trace_block", Traces::block_traces);
 		delegate.add_method("trace_call", Traces::call);
+		delegate.add_method("trace_sendRawTransaction", Traces::send_raw_transaction);
```

Comment:
a transaction != a call.

the naming follows conventions of the equivalent `eth_` methods; i.e. `eth_call` -> `trace_call`; `eth_sendRawTransaction` -> `trace_sendRawTransaction`.

that said, i'm quite ambivalent; `trace_rawTransaction` might well be sensible.